src/metrics/enhanced-metrics.ts

```diff
@@ -0,0 +1,26 @@
+import { getEnvValue, sendDistributionMetric } from "../index";
+
+import { parseTagsFromARN } from "../utils/arn";
+import { getColdStartTag } from "../utils/cold-start";
+
+const ENHANCED_LAMBDA_METRICS_NAMESPACE = "aws.lambda.enhanced";
+
+function areEnhancedMetricsEnabled() {
+  return getEnvValue("DD_ENHANCED_METRICS", "false").toLowerCase() === "true";
```

Comment:
Nice work with most of these changes. One small thing, the other config options are usually exposed both as an environment variable and a [config property](https://github.com/DataDog/datadog-lambda-layer-js/blob/2e6ee435fd009896384184cce3dfa40f68b75fe5/src/index.ts#L30) which we resolve in index.ts . Would you be able to move this logic there?